nano/node/socket.cpp

```diff
@@ -276,22 +271,45 @@ void nano::server_socket::on_connection (std::function<bool (std::shared_ptr<nan
 			}
 			else
 			{
-				this_l->node.logger.try_log ("Network: Unable to accept connection: ", ec_a.message ());
+				this_l->node.logger.always_log ("Network: Unable to accept connection: ", ec_a.message ());
 			}
 
-			// If the callback returns true, keep accepting new connections
-			if (callback_a (new_connection, ec_a))
+			if (this_l->is_temporary_error (ec_a))
 			{
 				this_l->on_connection (callback_a);
+				return;
 			}
-			else
+
+			if (this_l->is_system_error (ec_a))
```

Comment:
If it's not considered temporary, is it not better to let the listener know so it can recreate it?

Reply:
I think these are generic enough that they should be handled in here. It will result in the logs getting spammed with Network: Unable to accept connection: system limit hit so will poke the node handler in the right direction to resolve.

Reply:
OK